docs/api/ReactWrapper/hasClass.md

```diff
@@ -5,22 +5,28 @@ Returns whether or not the wrapped node has a `className` prop including the pas
 
 #### Arguments
 
-1. `className` (`String`): A single class name.
+1. `className` (`String` | `RegExp`): A single class name or a regex expression.
```

Comment:
Let's change the docs for ShallowWrapper too.

Reply:
Done!